data-prepper-plugins/blocking-buffer/README.md

```diff
@@ -0,0 +1,20 @@
+# Blocking Buffer
+
+This is a buffer based off `LinkedBlockingQueue` bounded to the specified capacity. One can read and write records with specified timeout value.
+
+## Configuration
+- buffer_size => An `int` representing max number of records the buffer accepts.
```

Comment:
Please also specify the default value. Same below for `batch_size`.

Reply:
Due to the addition of checkpoint API, here we need to specify that it represents the maximum number of unchecked records the buffer accepts, where
```
# of unchecked records = # of records written into the buffer + # of in-flight records (i.e. records read from the buffer but not yet checked by invoking checkpoint API).
```

Reply:
Will add this